src/Microsoft.PowerShell.Commands.Utility/commands/utility/EnableDisableRunspaceDebugCommand.cs

```diff
@@ -351,7 +351,7 @@ public SwitchParameter BreakAll
         }
```

Comment:
@KirkMunro, your last commit had 2 failures in `PowerShell-CI-linux`
ActionPreference.Break tests.-ErrorAction Break should break on a non-terminating error.The second statement should be the statement after that which generated the non-terminating error
```powershell
Expected 9, but got 7.
at <ScriptBlock>, /home/vsts/work/1/s/test/tools/Modules/HelpersDebugger/HelpersDebugger.psm1: line 189
189:         $extent.StartLineNumber | Should -Be $(if ($PSCmdlet.ParameterSetName -eq 'SingleLineExtent') {$Line} else {$FromLine})
```
ActionPreference.Break tests.-ErrorAction Break should break on a non-terminating error.The third statement should be the statement in the finally block
```powershell
Expected 13, but got 11.
at <ScriptBlock>, /home/vsts/work/1/s/test/tools/Modules/HelpersDebugger/HelpersDebugger.psm1: line 189
189:         $extent.StartLineNumber | Should -Be $(if ($PSCmdlet.ParameterSetName -eq 'SingleLineExtent') {$Line} else {$FromLine})
```